src/lib/core/fiber.h

```diff
@@ -625,6 +625,11 @@ struct fiber {
 			 * Should not be used in other fibers.
 			 */
 			struct lua_State *stack;
+			/**
+			 * Optional reference to userdata
+			 * represented current fiber id in Lua.
```

Comment:
Optional reference to userdata represent**ing** this fiber in Lua.

Reply:
Fixed. Thanks.